src/app/authentication/register/register.component.scss

```diff
@@ -0,0 +1,78 @@
+.btn-custom {
+    border: 1px solid #ed1c24;
+    background-color: rgba(255, 255, 255, 0.6);
+    border-radius: 0px;
+    color: #333;
+    font-size: 15px;
+    font-family: 'Roboto', sans-serif;
+    font-weight: normal;
+    text-transform: uppercase;
+    outline: none;
+    padding: 4px 36px 2px;
+}
+
+.sign-in-page {
+    margin-top: 150px;
+    background-color: #fff;
+    border: 1px #e5e5e5 solid;
+    border-bottom: 2px #e5e5e5 solid;
+    padding: 20px;
+    overflow: hidden;
+}
+
+.move-left {
+    margin-left: 16.667%;
+    width: 66.667%;
+}
+
+.create-new-account h4 {
+    font-size: 18px;
+    font-family: 'Open Sans', sans-serif;
+    padding-bottom: 14px;
+    border-bottom: 1px solid #ddd;
+}
+
+.create-new-account p {
+    font-size: 15px;
+    color: #666;
+}
```

Comment:
S obzirom da koristimo scss, probaj nesto ovako da li ce da radi. Nisam sad siguran napamet, ali otprilike:

`
.create-new-account {
  h4 {
    font-size: 18px;
    font-family: 'Open Sans', sans-serif;
    padding-bottom: 14px;
    border-bottom: 1px solid #ddd;
  }

  p {
    font-size: 15px;
    color: #666; 
  }
}
`

To je predno scssa, da mozes da ugnezdis ovako neke stvari koje su vezane za istu klasu ili isti element.
Recimo mozes i suprotno da odradis, da na jednom mestu izdvojis samo p, i onda tu za svaku klasu da ga stilizujes.
Kasnije kada budes gledao ili nagomilao razlicite paragrafe ili headinge, bice ti lakse da znas da su na jednom mestu

Reply:
Ne znam sad da li mora &h4 ili samo h4, ali probaj pa ces provaliti.
Mozda je bolje da se izdvoji po elementima, da umesto klasa { &element, bude element { &klasa.
Npr

```
h4 {
  .create-new-account {
  ...
  }
  .neka-druga-klasa {
  ...
  }
}
```

Reply:
Nismo mi nikad radili scss pa nisam ni znao koje su njegove prednosti, ali vazi, probacu to da promenim